### ISSUE TITLE
Impossible to run alongside darkflow

### ISSUE THREAD

**Issue by Sequential-circuits** [Visual-Behavior/detr-tensorflow#6]
I tried to run your system and darkflow in the same server

Too complicated: one needs to install 2 versions of CUDA and CUDNN and that has to be installed using .sh files instead of .pkg as the latter overwrites the initial system

And one needs to create a virtualenv to run one of those systems

And if the ubuntu is version 20 then it comes with python 3.85 which does not bring up tensorflow 1.14 required by darkflow, as it only brings versions above 2.0, so one needs to also install python 3.7 and hope it comes with pip 3.7

So it'd be nice if your system could identify small objects so one can move away from darkflow as that is just obsolete

**Comment by thibo73800** [comment]
Yes agree on small objects. I just added an issue to discuss the progress on this feature/implementation: https://github.com/Visual-Behavior/detr-tensorflow/issues/9
